direct.go

```diff
@@ -389,12 +478,16 @@ func (d *direct) dialServerWith(m *Masquerade) (net.Conn, error) {
 	tlsConfig := d.tlsConfig(m)
 	dialTimeout := 10 * time.Second
 	sendServerNameExtension := false
+	addr := m.IpAddress
+	if strings.IndexByte(addr, ':') == -1 {
```

Comment:
It would be safer to use `net.SplitHostAndPort`. IPv6 address for example actually contain colons.